frontend/src/components/views.js

```diff
@@ -848,24 +872,25 @@ class SchemeDetail extends React.Component {
                             </ListItem>
 
                             {flavors.map((function (f) {
-                                const f_paramsets = queryAll(db, "SELECT name FROM paramset WHERE flavor_id=?", [f.id]);
-                                const f_implementations = queryAll(db, "SELECT name FROM implementation WHERE flavor_id=?", [f.id]);
-                                const f_links = queryAll(db, "SELECT * FROM flavor_link WHERE flavor_id=?", [f.id]);
-                                const f_sources = queryAll(db, "SELECT * FROM flavor_source WHERE flavor_id=?", [f.id]);
+                                const f_paramsets = queryAll(db, "SELECT * FROM paramset WHERE flavor_id=? ORDER BY security_level_nist_category ASC, security_level_quantum ASC", [f.id]);
+                                const f_implementations = queryAll(db, "SELECT name FROM implementation WHERE flavor_id=? ORDER BY type DESC", [f.id]);
 
                                 return [
                                     <ListItem key={"flavor-" + f.id + "-head"} style={{ display: "block" }}>
-                                        <Typography component="h3" variant="h5">{f.name}</Typography>
+                                        <Typography component="h3" variant="h5">
+                                            <Link href={"?_=" + s.type + "/" + s.id + "/" + f.id}>{f.name}</Link>
```

Comment:
This should not use `href` but instead `component={RouterLink}` and `to` as for instance in line 247.

Reply:
Maybe you can add a wrapper analogous to `linkify` for internal links.